jis-documentation/src/test/java/BlueFilterTest.java

```diff
@@ -1,6 +1,6 @@
 import org.junit.Test;
```

Comment:
could we move all filter test to appropriate package like pl.edu.uj.JImageStream.tests.filters ?